.github/workflows/go.yml

```diff
@@ -54,23 +54,23 @@ jobs:
         include:
           - arch-label: AMD64
             arch: amd64
-            go: 1.17
+            go: 1.19
             runs-on: ubuntu-latest
-            staticcheck: v0.2.2
+            staticcheck: v0.4.5
```

Comment:
It seems that we always use staticcheck v0.4.5. Can we remove this parameter?

Reply:
Yea, we only needed it before because you couldn't use `latest` with go1.17, you had to use v0.2.2, and then we wanted to use the newer versions with go1.18. Now we can use the latest version with go1.19 and go1.20 but instead of using `latest` it's better to actually pin at v0.4.5. I've removed it from the parameters and just explicitly put it